config/checkstyle/suppressions.xml

```diff
@@ -13,6 +13,7 @@
     <suppress checks="AvoidEscapedUnicodeCharacters" files="src/main/java/org/openstreetmap/atlas/utilities/command/terminal/TTYAttribute.java" />
     <suppress checks="ParameterName" files="src/main/java/org/openstreetmap/atlas/utilities/function/TernaryFunction.java" />
     <suppress checks="ParameterName" files="src/main/java/org/openstreetmap/atlas/utilities/function/TernaryOperator.java" />
+    <suppress checks="ParameterName" files="src/main/java/org/openstreetmap/atlas/utilities/function/TernaryConsumer.java" />
```

Comment:
As I was commenting about checkstyle and the parameter names, this appeared hahaha

Reply:
Maybe we should just call them `one` `two` and `three` and not have to do that every time?

Reply:
Wouls still be nice to get rid of the suppressions, though..